include/seqan/file/file_mapping.h

```diff
@@ -101,7 +101,7 @@ enum FileMappingMode {
  * @brief The address range in the advise will not be needed any more.
  */
 
```

Comment:
concerning the entire filemapping I will speak with rene whether that is compiler or stdlib